source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -1094,20 +1100,16 @@ def create_next_kraken_request(self, request, responses):
         to do that we find ask the next (resp previous) query datetime
         """
 
-        # If Kraken send a new request date time, we use it
-        # for the next call to skip current Journeys
-        if responses and responses[0].HasField('next_request_date_time'):
-            request['datetime'] = responses[0].next_request_date_time
```

Comment:
(if this line can be removed, there may be some dead codes in kraken... can be handled later)

Reply:
You can't remove that, it will break the min_nb_journeys passed to kraken optimization.

Please add a test to check we don't break this feature.